leader/leader.go

```diff
@@ -182,6 +182,14 @@ func Become(ctx context.Context, lockName string, opts ...Option) error {
 					if err != nil {
 						log.Error(err, "Leader pod could not be deleted.")
 					}
+				case isNotReadyNode(ctx, config.Client, leaderPod.Spec.NodeName):
+					log.Info("the status of the node where operator pod with leader lock was running has been 'notReady'")
+					log.Info("Deleting the leader.")
+
+					//Mark the termainating status to the leaderPod and Delete the configmap lock
```

Comment:
Nit: should have a space after `//` : `// Mark `
Nit: typo `termainating` should be `terminating`.